pkgs/applications/file-managers/mucommander/default.nix

```
@@ -1,86 +1,108 @@
-{ lib, stdenv, fetchFromGitHub, gradle_6, perl, makeWrapper, jdk11, gsettings-desktop-schemas }:
+{ lib
+, stdenv
+, fetchFromGitHub
+, gradle_7
+, perl
+, makeWrapper
+, writeText
+, jdk11
+, gsettings-desktop-schemas
+}:
 
 let
-  version = "0.9.3-3";
+  version = "1.1.0-1";
 
   src = fetchFromGitHub {
     owner = "mucommander";
     repo = "mucommander";
     rev = version;
-    sha256 = "1zhglsx3b5k6np3ppfkkrqz9wg0j7ip598xxfgn75gjl020w0can";
+    sha256 = "sha256-sCBbY3aBSuJmyOuy36pg8X2jX6hXwW8SW2UzYyp/isM=";
   };
 
   postPatch = ''
     # there is no .git anyway
     substituteInPlace build.gradle \
-      --replace "git = org.ajoberstar.grgit.Grgit.open(file('.'))"  "" \
-      --replace "revision = git.head().id"                          "revision = 'abcdefgh'" \
-      --replace "proguard.enabled =" "// proguard.enabled =" \
-      --replace "version = '0.9.4'" "version = '${version}'"
-
-    # disable gradle plugins with native code and their targets
-    perl -i.bak1 -pe "s#(^\s*id '.+' version '.+'$)#// \1#" build.gradle
-    perl -i.bak2 -pe "s#(.*)#// \1# if /^(buildscript|task portable|task nsis|task proguard|task tgz|task\(afterEclipseImport\)|launch4j|macAppBundle|buildRpm|buildDeb|shadowJar)/ ... /^}/" build.gradle
-
-    # fix source encoding
-    find . -type f -name build.gradle \
-      -exec perl -i.bak3 -pe "s#(repositories\.jcenter\(\))#
-                                \1
-                                compileJava.options.encoding = 'UTF-8'
-                                compileTestJava.options.encoding = 'UTF-8'
-                               #" {} \;
+      --replace "git = grgit.open(dir: project.rootDir)"  "" \
+      --replace "revision = git.head().id" "revision = 'abcdefgh'"
```

```suggestion
      --replace "revision = git.head().id" "revision = '${version}'"
```
lets do this instead